docker/CMSRucioClient/examples/rucio-int-oidc.cfg

```diff
@@ -0,0 +1,13 @@
+[common]
```

Comment:
This file is not used, right? Just an example for someone who does want to authenticate by OIDC? 

I'd suggest changing this to the production server or adding rucio-oidc.cfg which uses the production server.

Reply:
yes, it's not used. I'm adding rucio-oidc.cfg in there.